src/arrayReverse.test.js

```diff
@@ -7,14 +7,29 @@ describe(`Function 'arrayReverse':`, () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
+  it(`should return an array of reversed strings in reversed order`, () => {
+    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+  });
 
+  it(`should return an empty array
+    if original array is empty`, () => {
+    expect(arrayReverse([])).toEqual([]);
   });
 
-  it(`should return an empty string
-    if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+  it(`should return an array of reversed strings and their order including special symbols`, () => {
+    expect(arrayReverse(['&Mate', 'Academy'])).toEqual(['ymeda', 'cAetaM&']);
+  });
+
+  it(`should return an array of reversed strings and their order including special numbers`, () => {
+    expect(arrayReverse(['8Mate', 'Academy'])).toEqual(['ymeda', 'cAetaM8']);
```

Comment:
The expected result should be `['ymedacA', '8etaM']` to reflect both the reversed strings and their order.